test/jumpTracker.test.ts

```diff
@@ -358,19 +369,22 @@ suite('Record and navigate jumps', () => {
     });
 
     suite('Can shifts jump lines up after deleting a line with Visual Line Mode', () => {
-      newJumpTest({
+      // TODO: this fails on Windows
+      newJumpTestSkipOnWindows({
         start: ['|start', 'a1', 'a2', 'a3', 'a4', 'a5', 'end'],
         keysPressed: '/^\nnnnkkdd',
         end: ['start', 'a1', '|a3', 'a4', 'a5', 'end'],
         jumps: ['start', 'a1', 'a3'],
       });
-      newJumpTest({
+      // TODO: this fails on Windows
+      newJumpTestSkipOnWindows({
         start: ['|start', 'a1', 'a2', 'a3', 'a4', 'a5', 'end'],
         keysPressed: '/^\nnnnkdd',
         end: ['start', 'a1', 'a2', '|a4', 'a5', 'end'],
         jumps: ['start', 'a1', 'a2', 'a4'],
       });
-      newJumpTest({
+      // TODO: this fails on Windows
```

Comment:
Can you make these `// TODO(#4844): this fails on Windows` so it's associated with the right ticket?

Reply:
Done!